.github/workflows/release-notes.yml

```
@@ -3,33 +3,51 @@ name: "Release Notes automation"
 on:
   workflow_dispatch:
     inputs:
-        target_branch:
-          type: string
-          default: 'master'
-          description: Target branch
-          required: true
         starting_commit:
           type: string
-          description: Commit from which to start generating the release notes (non-inclusive)
+          description: Commit from which to start generating the Release Notes (non-inclusive)
           required: true
-
+        overwrite:
+          type: boolean
+          description: Overwrite the content of the release notes (typically needed for RC1 notes)
+          required: true
+          default: false
 jobs:
   main:
     name: Generate Release Notes
     runs-on: ubuntu-latest
     steps:
 
-    - name: Checkout target branch
+    - name: Checkout ${{ github.ref }} branch in ${{ github.repository }} repository.
       uses: actions/checkout@v3
       with:
-        ref: ${{ github.event.inputs.target_branch }}
         fetch-depth: '0'
 
     - name: Get version
       id: get-version
-      uses: JActions/maven-version@v1.0.1
+      run: "echo \"resolved-version=\
+        $(mvn
+          --file ./pom.xml
```

Watch out for the file path.

```suggestion
          --file ./kura/pom.xml
```